src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -2353,6 +2335,26 @@ private static ParameterSymbol GetParameter(int argIndex, MemberAnalysisResult r
             }
 
             return BetterResult.Neither;
+
+            // Returns the parameter type (considering params).
+            static TypeSymbol getParameterTypeAndRefKind(int i, MemberAnalysisResult result, ImmutableArray<ParameterSymbol> parameters, TypeWithAnnotations paramsElementTypeOpt, out RefKind parameter1RefKind)
```

Comment:
>paramsElementTypeOpt [](http://example.com/codeflow?start=157&length=20)

Is this always `result.DefinitionParamsElementTypeOpt`? If so, consider removing this parameter.